### PR TITLE
My capstone project

### PR DESCRIPTION
**Pull request by julzedz**
## AR Summit

In my capstone project, I created and [deployed](https://julzedz.github.io/Conference/) an online website for an Augmented Reality Art Summit in line with the Microverse capstone project requirements.

In [this loom video](https://loom.com/share/9a4aa3d2434c41fd9e68ba32600226f1), I present a demo of the project features.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+      <ul class="desktop_menu">
+        <li class="menu_item intro"><a href="about.html">About</a></li>
+        <li class="menu_item program">Program</li>
+        <li class="menu_item application">Join</li>
+        <li class="menu_item sponsor">Sponsor</li>
+        <li class="menu_item news">News</li>
+        <li class="menu_item presentation">
+          <a href="#">CC Campaign</a>
+        </li>
+      </ul>
```
Comment (ichala) [inline about.html]: - [ ] the desktop version of the about page should have a link to the home page kindly add it as you did for the mobile menu

File: about.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        <li class="mobile-item">Join</li>
+        <li class="mobile-item" id="pres"><a href="#">Presentation</a></li>
+      </ul>
+      <ul class="desktop_menu">
+        <li class="menu_item intro"><a href="about.html">About</a></li>
+        <li class="menu_item program">Program</li>
+        <li class="menu_item application">Join</li>
+        <li class="menu_item sponsor">Sponsor</li>
+        <li class="menu_item news">News</li>
+        <li class="menu_item presentation">
```
Comment (ichala) [inline about.html]: - [ ] the desktop navbar looks awesome, kindly try to reduce the spaces between the links so they look correctly aligned to the right as shown in the template 

image

File: about.html [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    <section id="logo-sctn">
+      <header class="logo-headr">
+        <h2>AR Art Summit 2022 Logo</h2>
+        <div class="orange-bar"></div>
+      </header>
+      <div class="logo-info">
+        <p>The logo of AR Art Summit 2022 was decided through the logo competition from 8 June to 7 July.</p>
+      </div>
+      <div id="logo-image">
+        <img src="imgs/ar-art-summit-2022-600x315.jpeg" alt="summit logo">
```
Comment (ichala) [inline about.html:85]: - [ ] kindly reduce the size of the image that represent summit logo to match the template design


image

File: about.html [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+  <footer>
+    <div class="footer-logo">
+      <img id="mobile-footer" src="imgs/cc-logo2.png" alt="summit logo">
+      <img src="imgs/logo_cc_w_147x35.png" id="desktop-footer" alt="">
+    </div>
+    <div class="footer-note">
+      <p>2022 Creative Commons Korea.</p>
+      <p>Some Rights Reserved.</p>
+    </div>
+  </footer>
```
Comment (ichala) [inline about.html]: - [ ] the about page footer looks awesome and perfect, kindly apply the same alignment for the index footer as well

> TIP: in general the content of the footer should be the same on all pages , so you can put the same layout and text of the about page in the footer of index page and the colors can stay the same white for index & dark for about 💯 

About Footer :
image

index footer : 
image

File: about.html [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+  <title>About</title>
+  <link rel="stylesheet" href="about.css">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="first-header">
+    <a href=""><img src="imgs/facebook.png" alt=""></a>
+    <a href=""><img src="imgs/twitter.png" alt=""></a>
```
Comment (ichala) [inline about.html]: - [ ] all of your pages should be accessible, currently, you have some empty  `alt` attributes kindly add meaningful alts for each `img` tag you have

File: about.html [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+        </div>
+        <div class="sponsors">
+          <img src="imgs/naver-removebg-preview.png" alt="">
+        </div>
+        <div class="sponsors">
+          <img src="imgs/kakao-removebg-preview.png" alt="">
+        </div>
+        <div class="sponsors">
+          <img src="imgs/airbnb-removebg-preview.png" alt="">
+        </div>
```
Comment (ichala) [inline about.html]: - [ ]  all of your pages should be accessible, currently, you have some empty alt attributes kindly add meaningful alts for each img tag you have

File: about.html [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+      </div>
+    </section>
+  </main>
+  <footer>
+    <div class="footer-logo">
+      <img id="mobile-footer" src="imgs/cc-logo2.png" alt="summit logo">
+      <img src="imgs/logo_cc_w_147x35.png" id="desktop-footer" alt="">
```
Comment (ichala) [inline about.html]: - [ ]  all of your pages should be accessible, currently, you have some empty alt attributes kindly add meaningful alts for each img tag you have

File: index.html [hunk 8]
```diff
@@ -0,0 +1,191 @@
...
+  <title>Art Summit Home Page</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="first-header">
+    <a href=""><img class="icon" src="imgs/facebook.png" alt=""></a>
+    <a href=""><img class="icon" src="imgs/twitter.png" alt=""></a>
```
Comment (ichala) [inline index.html]: - [ ]  all of your pages should be accessible, currently, you have some empty alt attributes kindly add meaningful alts for each img tag you have

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @julzedz , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No linters errors✅ 
- Good Design ✅ 

🛑Note:  Your Presentation video looks good, but please note that you need to highlight a piece of the code that you find interesting or proud of.

 ## Required Changes ♻️

- [ ] _**check the comments below**_


## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @julzedz, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congrats

## Good Highlights✔

- Requested changes have been implemented
- Good commit messages
- Adherence to GitHub flow
- Linter checks are passing

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**